README.md

```diff
@@ -65,7 +65,7 @@ _TODO: Make it work with latest, confirm if there are any issues._
 6. **Install `grpccache-gen`**:
 
    ```
-   go get -u sourcegraph.com/sqs/grpccache/grpccache-gen
+   go get -u sourcegraph.com/sourcegraph/grpccache/grpccache-gen
```

Comment:
This is great. The `sourcegraph` version is 4 commits ahead of the `sqs` one.

Perhaps we can remove the `sqs` version after this is merged, to avoid confusion between 2 similar repos? /cc @sqs